app/code/Magento/Inventory/Test/Integration/Source/DefaultSource.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Test\Integration\Source;
+
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\Inventory\Model\Source;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class DefaultSource
+ * @package Magento\Inventory\Test\Integration\Source
+ */
+class DefaultSource extends TestCase
+{
+    /**
+     * @var Source
+     */
+    private $source;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $this->source = Bootstrap::getObjectManager()->create(SourceRepositoryInterface::class);
+        $this->source = $this->source->get(1);
```

Comment:
`$this->source->get(1)` looks like as part of `testDefaultSourceExist` test
Maybe need to move from `setUp` (where only initialization should be placed) to corresponding test method